techLogger.js

```diff
@@ -41,7 +43,7 @@ winston.addColors(levelsConfig.colors);
 
 function _makeLogger(prefix) {
 
-    var logPrefix = "[" + (prefix ? prefix : _getCallerFile()) + "]";
+    const logPrefix = `[${(prefix ? prefix : _getCallerFile())}]`;
```

Comment:
May be replaced by:

```
function _makeLogger(prefix = _getCallerFile()) {

    const logPrefix = `[$(prefix)]`;
```

(Thx https://github.com/ryanmcdermott/clean-code-javascript)